sphinxcontrib/drawio.py

```diff
@@ -19,6 +19,25 @@
 logger = logging.getLogger(__name__)
 
 VALID_OUTPUT_FORMATS = ("png", "jpg", "svg")
+X_DISPLAY_NUMBER = 1
+
+
+def is_headless(config: Config):
+    if config.drawio_headless == "auto":
+        if platform.system() == "Windows":
```

Comment:
Should be `platform.system() != "Linux"`, because Xvfb shouldn't even be run on Mac either, alternatively, can be  `platform.system() == "Windows" or platform.system() == "Darwin"` (Also never needs to run on mac since macs arent supposed to run headless)

Reply:
Okay, will change.